src/smk/viewer-leaflet/tool/minimap/tool-minimap-leaflet.js

```diff
@@ -6,7 +6,7 @@ include.module( 'tool-minimap-leaflet', [ 'leaflet', 'tool-minimap' ], function
 
         smk.addToStatus( $( '<div class="smk-spacer">' ).height( 170 ).get( 0 ) )
 
-        var ly = smk.$viewer.createBasemapLayer( this.baseMap || "Topographic" );
+        var ly = smk.$viewer.createBasemapLayer( smk.viewer.esriApiKey, this.baseMap || "Topographic" );
```

Comment:
Same idea here as for the basemap tool, I don't think the minimap tool needs to be aware of the esriApiKey.